pymc3/distributions/continuous.py

```diff
@@ -1534,12 +1533,79 @@ def _repr_latex_(self, name=None, dist=None):
                                                                 get_variable_name(beta))
 
 
-def StudentTpos(*args, **kwargs):
-    warnings.warn("StudentTpos has been deprecated. In future, use HalfStudentT instead.",
-                DeprecationWarning)
-    return HalfStudentT(*args, **kwargs)
+class HalfStudentT(Continuous):
```

Comment:
Shouldn't we inherit from `PositiveContinuous` here?

Reply:
you are right, I totally missed that!

Reply:
fixed, thank you.